mappings/net/minecraft/entity/projectile/ProjectileEntity.mapping

```diff
@@ -30,11 +38,23 @@ CLASS net/minecraft/class_1676 net/minecraft/entity/projectile/ProjectileEntity
 		ARG 1 entity
 	METHOD method_7454 onEntityHit (Lnet/minecraft/class_3966;)V
 		ARG 1 entityHitResult
-	METHOD method_7485 setVelocity (DDDFF)V
+	METHOD method_7485 setProperties (DDDFF)V
```

Comment:
`setVelocity` imo is still better as the result is setting velocity; it's that the velocity is calculated in a somewhat quirky way.